### DISCUSSION TITLE
How to get galaxy brain badge ?

### DISCUSSION THREAD

**Discussion by rnik12** [Schweinepriester/github-profile-achievements#523] [General]
How to get galaxy brain badge ?

**Comment by jarek-pacocha** [comment]
Hi @rnik12!

To get a Galaxy brain badge you need to do the following twice:

1. Comment a topic in Discussions like this one.
2. Get your comment marked as an answer by an author. 

See if you can see a button to mark my comment as an answer below. 
Good luck! 👍

**Reply by aliencaocao** [reply]
certainly did not work for me: https://github.com/Schweinepriester/github-profile-achievements/discussions/535

**Reply by tom-mcIvor** [reply]
mark my reply here as an answer

**Reply by shambhaveesrivastava12** [reply]
hey @rnik12 can you mark my answer?

**Comment by jarek-pacocha** [comment]
Hey @rnik12 could you mark my response as an answer? I'd really appreciate that. Thanks! 👍🏻

**Reply by tom-mcIvor** [reply]
think you need to reply to someones post. this is an answer wink wink..

**Comment by HasanSafarli** [comment]
> How to get galaxy brain badge ?

how does this work?

**Comment by muhajiralfath** [comment]
Help me please

**Comment by cedev935** [comment]
If you receive Galaxy Brain Badge, you have to answer the question at least.
And if client click "Mark as read" button, you'll get Galaxy Badge.
Thanks

**Comment by chanmyaemaung** [comment]
Could you please mark my response as an answer? :D

**Comment by HarshDekate** [comment]
Thank you for the information! If we come across a suitable discussion topic, we'll be sure to participate and try to earn that Galaxy brain badge. 👍😄
